### PR TITLE
PEAR-2214 - Create Workspace for Shared Resources

### PR DESCRIPTION
**Pull request by amlehman**
## Description
\+ Added a small component to test the package was set up correctly 

## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Comment by wteouchicago** [comment]
@amlehman -- anything I can/should test here?

**Comment by amlehman** [comment]
@wteouchicago, nope! It's just setting up some config.

File: packages/portal-components/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@gff/portal-components",
+  "version": "1.0.0",
```
Comment (paribartandhakal) [inline packages/portal-components/package.json]: Do we want to align this version with other packages for ease? I noticed that the enclave portal was set to `2.15.5` in this [PR](https://github.com/NCI-GDC/gdc-frontend-framework/pull/1368)
cc. @craigrbarnes

Comment (amlehman) [reply packages/portal-components/package.json]: Yeah, it can just be the same as the other packages 👍

Comment (craigrbarnes) [reply packages/portal-components/package.json]: I agree

Comment (paribartandhakal) [reply packages/portal-components/package.json]: I noticed that 2.15.5 has been merged to develop.

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**